setup.py

```diff
@@ -15,7 +20,7 @@
     setup_requires=['setuptools_scm'],
     install_requires=['click', 'dask', 'docker', 'docker-map', 'toolz',
                       'buildbot', 'treq'],
-    tests_require=['pytest>=3.9'],
+    tests_require=['pytest>=3.9', 'mock'],
```

Comment:
Oh, I forgot: `mock` is available as [unittest.mock](https://docs.python.org/3/library/unittest.mock.html) in Python 3, you shouldn't need to install it separately.

Reply:
It was complaining somewhy, probably because `twisted.trial` requires it.

Reply:
Or buildbot itself: https://github.com/buildbot/buildbot/blob/master/master/setup.py#L487

Reply:
Created another issue: https://github.com/ursa-labs/ursabot/issues/30